pkg/webhook/handler.go

```diff
@@ -117,7 +117,11 @@ func handle(ctx context.Context, req admission.Request, r *http.Request, f mutat
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
 
-		return admission.PatchResponseFromRaw(oldObjMarshaled, newObjMarshaled)
+		patch := admission.PatchResponseFromRaw(oldObjMarshaled, newObjMarshaled)
+		logger.Info("Old", "oldObjMarshaled", string(oldObjMarshaled))
```

Comment:
Can you change the log level to debug? I don't think we want/need to see this every time.

Reply:
I removed it. Sorry for that.